tools/accuracy_checker/accuracy_checker/evaluators/custom_evaluators/text_spotting_evaluator.py

```diff
@@ -18,6 +18,7 @@
 from functools import partial
 from collections import OrderedDict
 import numpy as np
+from scipy.special import softmax
```

Comment:
could you please make scipy is optional requirement?
there is some trouble to install it on some target platforms where AC used for validation (i mean yocto now).
also as I know softmax was supported only in 1.2 release, it means that we need change lower bound version in requirements files.

Reply:
"there is some trouble to install it on some target platforms where AC used for validation (i mean yocto now)" 
What should I do with this info? Should I continue use scipy or what?

Reply:
is ok if I put it here https://github.com/openvinotoolkit/open_model_zoo/blob/develop/tools/accuracy_checker/requirements.in?

Reply:
First of all, I need to understand, is it possible to get the same result without scipy usage?
Decision to continue use scipy or not is up to you. 
but if you want to stay it, please look how we make deal with sklearn in this case
https://github.com/openvinotoolkit/open_model_zoo/blob/develop/tools/accuracy_checker/accuracy_checker/metrics/classification.py
It needs to not break validation other models on that system where probably your model currently does not work at all.

Reply:
I can implement softmax, but as far as I can see scipy is https://github.com/openvinotoolkit/open_model_zoo/blob/develop/tools/accuracy_checker/requirements.in already.

Reply:
removed scipy